src/features/isanteplus/patientSummary.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Patient Summary
+
+ Background:
+    Given system user logs into Isanteplus application and goes to the Home page
+
+  @patientSummary
+  Scenario: Patient Summary widget
+  When  system loads Patient dashboardPage
```

Comment:
can you follow the work flow desribed 


    1. Search for a patient (Use case #4.1 or 4.2)
    2. Select ‘‘Patient summary’ on the right hand side menu